<a href="https://colab.research.google.com/github/elmilanos/ProbabilityStatistics/blob/main/Pr%C3%A1ctica_M%C3%B3dulo_5_Probabilidad_Condicional_y_Teorema_de_Bayes.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Aplicacion del Teorema de Bayes

El Teorema de Bayes se usa para actualizar nuestras creencias basadas en evidencia.
Aquí, permite calcular la probabilidad de que una persona en Durango tenga la enfermedad respiratoria **dado que tiene tos**.

Este enfoque es útil en medicina para evaluar síntomas y enfermedades antes de hacer un diagnóstico definitivo.

In [1]:
# Importamos librerías necesarias
def teorema_de_bayes():

    # Calcula la probabilidad de que un paciente con tos tenga la enfermedad respiratoria en Durango
    # usando el Teorema de Bayes.


    # Probabilidades dadas en el problema
    P_A = 0.10  # Probabilidad de tener la enfermedad
    P_B_dado_A = 0.80  # Probabilidad de tener tos si se tiene la enfermedad
    P_B_dado_no_A = 0.20  # Probabilidad de tener tos si NO se tiene la enfermedad
    P_no_A = 1 - P_A  # Probabilidad de NO tener la enfermedad

    # Calcular la probabilidad total de tos (P(B))
    P_B = (P_B_dado_A * P_A) + (P_B_dado_no_A * P_no_A)

    # Aplicar el Teorema de Bayes para obtener P(A|B) (probabilidad de tener la enfermedad dado que tiene tos)
    P_A_dado_B = (P_B_dado_A * P_A) / P_B

    # Imprimir resultados
    print("RESULTADOS:")
    print(f"Probabilidad de que una persona con tos tenga la enfermedad: {P_A_dado_B:.4f} ({P_A_dado_B * 100:.2f}%)")

# Llamamos a la función
teorema_de_bayes()


RESULTADOS:
Probabilidad de que una persona con tos tenga la enfermedad: 0.3077 (30.77%)


# Aplicacion en Durango
Este análisis es clave para **tomar mejores decisiones en salud pública en Durango**, ya que:

- Permite **identificar qué tan confiable es la tos como indicador de la enfermedad.**
- Ayuda a **optimizar recursos**, evitando pruebas médicas innecesarias en pacientes de bajo riesgo.
- Puede ser usado en **políticas de salud**, como campañas para detectar otras enfermedades respiratorias.

## Resultados & Conclusiones
El resultado nos dice que **aproximadamente el 30.77% de los pacientes con tos tienen la enfermedad respiratoria**.
Esto significa que, aunque la tos es un síntoma común, en la mayoría de los casos no implica que el paciente tenga la enfermedad respiratoria grave.

## Conclusión:

- Es importante **realizar más pruebas** antes de diagnosticar una enfermedad solo con base en la tos.

- Este análisis ayuda a **tomar decisiones en salud pública**, como mejorar el acceso a pruebas médicas.